**Core -> Full Integration (with SecurityManager)** [Build 3](https://ci.wildfly.org/buildConfiguration/WildFlyCore_PullRequest32xWildFly40_WildFlyCoreFullIntegrationBootableJarJdk21/567351) outcome was **FAILURE** using a merge of 55afa87a5953a5d44960bdb15a82fee8fc7b24d6
Summary: Tests failed: 24 (24 new), passed: 4258, ignored: 49 Build time: 02:47:07
### Failed tests
```
org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testStartWithNonExistentAttributeYaml: java.lang.AssertionError: Server log must contain ERROR with information which attribute is wrong.
Expected: a string containing log parts in order: ["WFLYSRV0055", "WFLYCTL0509", "non-existent-attribute", "/socket-binding-group=standard-sockets"]
     but: was "Exception in thread \"main\" java.lang.IllegalStateException: The install directory /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core may still be in the process of being deleted. The marker file /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core/wildfly-cleanup-marker has not been deleted within 10s. Please check for a previous job running and delete the marker file if it was left behind because of an error.\n\tat org.wildfly.core.jar.boot.Main.waitForDelete(Main.java:342)\n\tat org.wildfly.core.jar.boot.Main.main(Main.java:128)\n"
java.lang.AssertionError: 
Server log must contain ERROR with information which attribute is wrong.
Expected: a string containing log parts in order: ["WFLYSRV0055", "WFLYCTL0509", "non-existent-attribute", "/socket-binding-group=standard-sockets"]
     but: was "Exception in thread \"main\" java.lang.IllegalStateException: The install directory /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core may still be in the process of being deleted. The marker file /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core/wildfly-cleanup-marker has not been deleted within 10s. Please check for a previous job running and delete the marker file if it was left behind because of an error.\n\tat org.wildfly.core.jar.boot.Main.waitForDelete(Main.java:342)\n\tat org.wildfly.core.jar.boot.Main.main(Main.java:128)\n"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testStartWithNonExistentAttributeYaml(YamlExtensionTestCase.java:432)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at org.wildfly.core.testrunner.WildFlyRunner$WrappedStatement.evaluate(WildFlyRunner.java:357)
	at org.wildfly.core.testrunner.WildFlyRunner.run(WildFlyRunner.java:198)
------- Stdout: -------


org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testRemoveNonExistentResource: java.lang.RuntimeException: Could not start container
	at org.wildfly.core.testrunner.Server.start(Server.java:325)
	at org.wildfly.core.testrunner.ServerController.start(ServerController.java:140)
	at org.wildfly.core.testrunner.ServerController.startYamlExtension(ServerController.java:183)
	at org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testRemoveNonExistentResource(YamlExtensionTestCase.java:477)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at org.wildfly.core.testrunner.WildFlyRunner$WrappedStatement.evaluate(WildFlyRunner.java:357)
	at org.wildfly.core.testrunner.WildFlyRunner.run(WildFlyRunner.java:198)
Caused by: java.lang.RuntimeException: Managed server was not started within 60 seconds.
	at org.wildfly.core.testrunner.Server.start(Server.java:320)
	... 44 more
------- Stdout: -------


org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testUndefineNonExistentAttributeYamlOperations: java.lang.AssertionError: Server log must contain the correct error IDs and attribute name.
Expected: a string containing log parts in order: ["WFLYCTL0013", "undefine-attribute", "WFLYCTL0201", "asfds"]
     but: was "Exception in thread \"main\" java.lang.IllegalStateException: The install directory /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core may still be in the process of being deleted. The marker file /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core/wildfly-cleanup-marker has not been deleted within 10s. Please check for a previous job running and delete the marker file if it was left behind because of an error.\n\tat org.wildfly.core.jar.boot.Main.waitForDelete(Main.java:342)\n\tat org.wildfly.core.jar.boot.Main.main(Main.java:128)\n"
java.lang.AssertionError: 
Server log must contain the correct error IDs and attribute name.
Expected: a string containing log parts in order: ["WFLYCTL0013", "undefine-attribute", "WFLYCTL0201", "asfds"]
     but: was "Exception in thread \"main\" java.lang.IllegalStateException: The install directory /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core may still be in the process of being deleted. The marker file /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core/wildfly-cleanup-marker has not been deleted within 10s. Please check for a previous job running and delete the marker file if it was left behind because of an error.\n\tat org.wildfly.core.jar.boot.Main.waitForDelete(Main.java:342)\n\tat org.wildfly.core.jar.boot.Main.main(Main.java:128)\n"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testUndefineNonExistentAttributeYamlOperations(YamlExtensionTestCase.java:458)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at org.wildfly.core.testrunner.WildFlyRunner$WrappedStatement.evaluate(WildFlyRunner.java:357)
	at org.wildfly.core.testrunner.WildFlyRunner.run(WildFlyRunner.java:198)
------- Stdout: -------


org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testAddingExtensionPathDeploymentOverlayYamlLogsWarnings: java.lang.RuntimeException: Could not start container
	at org.wildfly.core.testrunner.Server.start(Server.java:325)
	at org.wildfly.core.testrunner.ServerController.start(ServerController.java:140)
	at org.wildfly.core.testrunner.ServerController.startYamlExtension(ServerController.java:183)
	at org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testAddingExtensionPathDeploymentOverlayYamlLogsWarnings(YamlExtensionTestCase.java:246)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at org.wildfly.core.testrunner.WildFlyRunner$WrappedStatement.evaluate(WildFlyRunner.java:357)
	at org.wildfly.core.testrunner.WildFlyRunner.run(WildFlyRunner.java:198)
Caused by: java.lang.RuntimeException: Managed server was not started within 60 seconds.
	at org.wildfly.core.testrunner.Server.start(Server.java:320)
	... 44 more
------- Stdout: -------


org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testStartWithBadlyIndentedYaml: java.lang.AssertionError: Server must provide useful information where yaml is wrong.
Expected: a string containing log parts in order: ["mapping values are not allowed here", "port-offset"]
     but: was "Exception in thread \"main\" java.lang.IllegalStateException: The install directory /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core may still be in the process of being deleted. The marker file /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core/wildfly-cleanup-marker has not been deleted within 10s. Please check for a previous job running and delete the marker file if it was left behind because of an error.\n\tat org.wildfly.core.jar.boot.Main.waitForDelete(Main.java:342)\n\tat org.wildfly.core.jar.boot.Main.main(Main.java:128)\n"
java.lang.AssertionError: 
Server must provide useful information where yaml is wrong.
Expected: a string containing log parts in order: ["mapping values are not allowed here", "port-offset"]
     but: was "Exception in thread \"main\" java.lang.IllegalStateException: The install directory /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core may still be in the process of being deleted. The marker file /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core/wildfly-cleanup-marker has not been deleted within 10s. Please check for a previous job running and delete the marker file if it was left behind because of an error.\n\tat org.wildfly.core.jar.boot.Main.waitForDelete(Main.java:342)\n\tat org.wildfly.core.jar.boot.Main.main(Main.java:128)\n"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testStartWithBadlyIndentedYaml(YamlExtensionTestCase.java:418)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at org.wildfly.core.testrunner.WildFlyRunner$WrappedStatement.evaluate(WildFlyRunner.java:357)
	at org.wildfly.core.testrunner.WildFlyRunner.run(WildFlyRunner.java:198)
------- Stdout: -------


org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testAttributeOverrideByTwoYamlFiles: java.lang.RuntimeException: Could not start container
	at org.wildfly.core.testrunner.Server.start(Server.java:325)
	at org.wildfly.core.testrunner.ServerController.start(ServerController.java:140)
	at org.wildfly.core.testrunner.ServerController.startYamlExtension(ServerController.java:183)
	at org.wildfly.core.testrunner.ServerController.startYamlExtension(ServerController.java:179)
	at org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testAttributeOverrideByTwoYamlFiles(YamlExtensionTestCase.java:394)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at org.wildfly.core.testrunner.WildFlyRunner$WrappedStatement.evaluate(WildFlyRunner.java:357)
	at org.wildfly.core.testrunner.WildFlyRunner.run(WildFlyRunner.java:198)
Caused by: java.lang.RuntimeException: Managed server was not started within 60 seconds.
	at org.wildfly.core.testrunner.Server.start(Server.java:320)
	... 45 more
------- Stdout: -------
Exception in thread "main" java.lang.IllegalStateException: The install directory /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core may still be in the process of being deleted. The marker file /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core/wildfly-cleanup-marker has not been deleted within 10s. Please check for a previous job running and delete the marker file if it was left behind because of an error.
	at org.wildfly.core.jar.boot.Main.waitForDelete(Main.java:342)
	at org.wildfly.core.jar.boot.Main.main(Main.java:128)


org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testReplacingResourceByEmptyResourceLogsWarning: java.lang.RuntimeException: Could not start container
	at org.wildfly.core.testrunner.Server.start(Server.java:325)
	at org.wildfly.core.testrunner.ServerController.start(ServerController.java:140)
	at org.wildfly.core.testrunner.ServerController.startYamlExtension(ServerController.java:183)
	at org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testReplacingResourceByEmptyResourceLogsWarning(YamlExtensionTestCase.java:404)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at org.wildfly.core.testrunner.WildFlyRunner$WrappedStatement.evaluate(WildFlyRunner.java:357)
	at org.wildfly.core.testrunner.WildFlyRunner.run(WildFlyRunner.java:198)
Caused by: java.lang.RuntimeException: Managed server was not started within 60 seconds.
	at org.wildfly.core.testrunner.Server.start(Server.java:320)
	... 44 more
------- Stdout: -------


org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testEmptyExtensionInYamlLogsWarnings: java.lang.RuntimeException: Could not start container
	at org.wildfly.core.testrunner.Server.start(Server.java:325)
	at org.wildfly.core.testrunner.ServerController.start(ServerController.java:140)
	at org.wildfly.core.testrunner.ServerController.startYamlExtension(ServerController.java:183)
	at org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testEmptyExtensionInYamlLogsWarnings(YamlExtensionTestCase.java:261)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at org.wildfly.core.testrunner.WildFlyRunner$WrappedStatement.evaluate(WildFlyRunner.java:357)
	at org.wildfly.core.testrunner.WildFlyRunner.run(WildFlyRunner.java:198)
Caused by: java.lang.RuntimeException: Managed server was not started within 60 seconds.
	at org.wildfly.core.testrunner.Server.start(Server.java:320)
	... 44 more
------- Stdout: -------


org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testIdempotence: java.lang.RuntimeException: Could not start container
	at org.wildfly.core.testrunner.Server.start(Server.java:325)
	at org.wildfly.core.testrunner.ServerController.start(ServerController.java:140)
	at org.wildfly.core.testrunner.ServerController.startYamlExtension(ServerController.java:183)
	at org.wildfly.core.testrunner.ServerController.startYamlExtension(ServerController.java:179)
	at org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testIdempotence(YamlExtensionTestCase.java:536)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at org.wildfly.core.testrunner.WildFlyRunner$WrappedStatement.evaluate(WildFlyRunner.java:357)
	at org.wildfly.core.testrunner.WildFlyRunner.run(WildFlyRunner.java:198)
Caused by: java.lang.RuntimeException: Managed server was not started within 60 seconds.
	at org.wildfly.core.testrunner.Server.start(Server.java:320)
	... 45 more
------- Stdout: -------
Exception in thread "main" java.lang.IllegalStateException: The install directory /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core may still be in the process of being deleted. The marker file /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core/wildfly-cleanup-marker has not been deleted within 10s. Please check for a previous job running and delete the marker file if it was left behind because of an error.
	at org.wildfly.core.jar.boot.Main.waitForDelete(Main.java:342)
	at org.wildfly.core.jar.boot.Main.main(Main.java:128)


org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testReadConfigAsXmlFile: java.lang.RuntimeException: Could not start container
	at org.wildfly.core.testrunner.Server.start(Server.java:325)
	at org.wildfly.core.testrunner.ServerController.start(ServerController.java:140)
	at org.wildfly.core.testrunner.ServerController.startYamlExtension(ServerController.java:183)
	at org.wildfly.core.testrunner.ServerController.startYamlExtension(ServerController.java:179)
	at org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testReadConfigAsXmlFile(YamlExtensionTestCase.java:512)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at org.wildfly.core.testrunner.WildFlyRunner$WrappedStatement.evaluate(WildFlyRunner.java:357)
	at org.wildfly.core.testrunner.WildFlyRunner.run(WildFlyRunner.java:198)
Caused by: java.lang.RuntimeException: Managed server was not started within 60 seconds.
	at org.wildfly.core.testrunner.Server.start(Server.java:320)
	... 45 more
------- Stdout: -------
Exception in thread "main" java.lang.IllegalStateException: The install directory /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core may still be in the process of being deleted. The marker file /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core/wildfly-cleanup-marker has not been deleted within 10s. Please check for a previous job running and delete the marker file if it was left behind because of an error.
	at org.wildfly.core.jar.boot.Main.waitForDelete(Main.java:342)
	at org.wildfly.core.jar.boot.Main.main(Main.java:128)


org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testAddingResourceWithOverridingAttributesByTwoYamlFiles: java.lang.RuntimeException: Could not start container
	at org.wildfly.core.testrunner.Server.start(Server.java:325)
	at org.wildfly.core.testrunner.ServerController.start(ServerController.java:140)
	at org.wildfly.core.testrunner.ServerController.startYamlExtension(ServerController.java:183)
	at org.wildfly.core.testrunner.ServerController.startYamlExtension(ServerController.java:179)
	at org.jboss.as.test.manualmode.management.persistence.yaml.YamlExtensionTestCase.testAddingResourceWithOverridingAttributesByTwoYamlFiles(YamlExtensionTestCase.java:379)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at org.wildfly.core.testrunner.WildFlyRunner$WrappedStatement.evaluate(WildFlyRunner.java:357)
	at org.wildfly.core.testrunner.WildFlyRunner.run(WildFlyRunner.java:198)
Caused by: java.lang.RuntimeException: Managed server was not started within 60 seconds.
	at org.wildfly.core.testrunner.Server.start(Server.java:320)
	... 45 more
------- Stdout: -------
Exception in thread "main" java.lang.IllegalStateException: The install directory /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core may still be in the process of being deleted. The marker file /opt/buildAgent/work/58b81e6fd13b3b53/core/testsuite/manualmode/target/wildfly-core/wildfly-cleanup-marker has not been deleted within 10s. Please check for a previous job running and delete the marker file if it was left behind because of an error.
	at org.wildfly.core.jar.boot.Main.waitForDelete(Main.java:342)
	at org.wildfly.core.jar.boot.Main.main(Main.java:128)



##### there are 14 more failed tests, see build details
```

--- a/elytron/src/main/java/org/wildfly/extension/elytron/_private/ElytronSubsystemMessages.java
+++ b/elytron/src/main/java/org/wildfly/extension/elytron/_private/ElytronSubsystemMessages.java
@@ -671,7 +671,7 @@ public interface ElytronSubsystemMessages extends BasicLogger {
     @LogMessage(level = WARN)
     void noAllowedJkuValuesSpecifiedForTokenRealm(String realmName, String systemPropertyName);
 
-    @Message(id = 1091, value = "From Jakarta EE 11 setting a Policy is no longer required, the /subsystem=elytron/policy=* resource(s) can be removed.")
+    @Message(id = 1091, value = "In an EE 11 or later environment setting a Policy is no longer required; the /subsystem=elytron/policy=* resource(s) can be removed.")
     @LogMessage(level = WARN)
     void settingPolicyNotSupported();
 

--- a/elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties
+++ b/elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties
@@ -1663,7 +1663,7 @@ elytron.dir-context.properties=The additional connection properties for the DirC
 #######################
 
 elytron.policy=A definition that sets up a policy provider.
-elytron.policy.deprecated=Support for security Policy implementations has been removed from Java 25, setting a JACC Policy will no longer be required from Jakarta EE 11.
+elytron.policy.deprecated=Support for security Policy implementations has been removed from Java 25, and setting a JACC Policy is no longer required in an EE 11 or later environment.
 # Operations
 elytron.policy.add=Add operation for adding a policy provider.
 elytron.policy.remove=Remove operation for removing a policy provider.

